project/perform-release.sh

```diff
@@ -9,10 +9,15 @@ if [[ $1 != "patch" && $1 != "minor" && $1 != "major" ]] ; then
   exit 1;
 fi
 
-echo ""
-echo "Sign into npm"
-echo ""
-npm whoami &>/dev/null || npm login
+currentBranch="$(git symbolic-ref HEAD 2>/dev/null)" ||
+currentBranch="(unnamed currentBranch)"     # detached HEAD
+
+currentBranch=${currentBranch##refs/heads/}
```

Comment:
It was referenced on a S/O post and I couldn't find a better option

Happy to change this for anything easier / more obvious.

On Wed, 16 Mar 2016, 13:26 Mat Scales, <notifications@github.com> wrote:

> In project/perform-release.sh
> <https://github.com/GoogleChrome/Propel/pull/52#discussion_r56331688>:
>
> > @@ -9,10 +9,15 @@ if [[ $1 != "patch" && $1 != "minor" && $1 != "major" ]] ; then
> >    exit 1;
> >  fi
> >
> > -echo ""
> > -echo "Sign into npm"
> > -echo ""
> > -npm whoami &>/dev/null || npm login
> > +currentBranch="$(git symbolic-ref HEAD 2>/dev/null)" ||
> > +currentBranch="(unnamed currentBranch)"     # detached HEAD
> > +
> > +currentBranch=${currentBranch##refs/heads/}
>
> Whaaaat? What is this magic? Do you have a link to docs for this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/Propel/pull/52/files/f7ce773dcd6ee475a3f04b37a5c40cadd969f13f#r56331688>
>